xtest/runner.py

```diff
@@ -55,19 +55,25 @@ def run_cli_tests(sdks_encrypt, sdks_decrypt, pt_file):
     if not sdks_encrypt or not sdks_decrypt:
         return
     logger.info("--- run_cli_tests %s => %s", sdks_encrypt, sdks_decrypt)
-    fail = []
+    tdf3fail = []
+    nanofail = []
 
     serial = 0
     for x in sdks_encrypt:
         for y in sdks_decrypt:
             try:
                 test_cross_roundtrip(x, y, serial, pt_file)
             except Exception as e:
-                logger.error("Exception with pass %s => %s", x, y, exc_info=True)
-                fail += [f"{x}=>{y}"]
+                logger.error("Exception with ZTDF pass %s => %s", x, y, exc_info=True)
```

Comment:
I think "ZTDF" is proprietary which this is not testing

Reply:
made that update based on dave's comment's above